zstacklib/zstacklib/utils/lvm.py

```
@@ -2743,3 +2744,210 @@ def check_lvmlockd_log():
+def update_vg_tag(vg_uuid, host_uuid):
+    """Update VG INIT_TAG: replace source platform creator info with current platform."""
+    r, o, e = bash.bash_roe("vgs --nolocking -t -o tags --noheadings %s" % vg_uuid)
+    if r != 0:
+        logger.warn("failed to get VG tags for %s, skipping tag update" % vg_uuid)
+        return
+
+    old_tag = o.strip()
+    if not old_tag:
+        return
+
+    new_tag = "%s::%s::%s::%s" % (INIT_TAG, host_uuid, time.time(), linux.get_hostname())
```

Ruff (0.15.5)

[error] Undefined name `INIT_TAG`

(F821)